knexfile.js

```diff
@@ -0,0 +1,49 @@
+// Update with your config settings.
+
+module.exports = {
+  development: {
+    client: "sqlite3",
+    connection: {
+      filename: "./data/todoList.sqlite3"
+    },
+    useNullAsDefault: true,
+    migrations: {
+      directory: "./data/migrations"
+    },
+    seeds: {
+      directory: "./data/seeds"
+    }
+  },
+
+  staging: {
+    client: "postgresql",
+    connection: {
+      database: "my_db",
+      user: "username",
+      password: "password"
+    },
+    pool: {
+      min: 2,
+      max: 10
+    },
+    migrations: {
+      tableName: "knex_migrations"
+    }
+  },
+
+  production: {
+    client: "postgresql",
+    connection: {
+      database: "my_db",
+      user: "username",
+      password: "password"
+    },
+    pool: {
+      min: 2,
+      max: 10
+    },
+    migrations: {
+      tableName: "knex_migrations"
+    }
+  }
```

Comment:
You could go ahead and get rid of your `staging` and `production` properties since all that you're currently using (and all that's currently functional) is the `development` configuration.